src/logic/PlayerController.cpp

```diff
@@ -835,8 +835,7 @@ void PlayerController::onUpdateByInput(float deltaTime)
             }
             else
             {
-                // This happens when NPC enters area that should not be accessible
```

Comment:
Trace level would have a compile switch to make it compile to nothing and you'd only use it when you're interested.